app/templates/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "moduleName",
+  "version": "0.0.0",
+  "description": "",
+  "license": "MIT",
+  "repository": "githubUsername/moduleName",
+  "author": {
+    "name": "name",
+    "email": "email",
+    "url": "website"
+  },
+  "engines": {
+    "node": ">=0.10.0"
+  },
+  "scripts": {
+		"start": "node index.js"
+  },
+  "keywords": [
+    ""
+  ],
+  "dependencies": {
+    "h5bp": "^0.1.2"
+  },
+  "devDependencies": {}
+}
```

Comment:
Since this is for a server, we can drop everything except for `scripts` and `dependencies`.